src/dreyfus_util.erl

```diff
@@ -166,6 +167,46 @@ time(Metric, {M, F, A}) when is_list(Metric) ->
         couch_stats:update_histogram([dreyfus | Metric],  Length)
     end.
 
+utc_timestamp() ->
```

Comment:
We should consider adding this function to couch_util since its used multiple times. We may also want to check if something like this already exists.